packages/freezed/test/integration/concrete.dart

```diff
@@ -96,3 +96,10 @@ class Const with _$Const {
   const Const._();
   const factory Const(int a) = _Const;
 }
+
+@freezed
+class Dollar with _$Dollar {
```

Comment:
We'd want to test $ in the middle/end of the variable name too.

And in constructor names

Reply:
> We'd want to test $ in the middle/end of the variable name too.
> 
> And in constructor names

@rrousselGit Added, now both named constructors and class names support interspersed $ characters